### ISSUE TITLE
Service Mark (SM) symbol is not announced by JAWS on IE

### ISSUE THREAD

**Issue by andrewPoblete** [FreedomScientific/standards-support#87]
### Summary

The Service Mark (SM) symbol is not announced by JAWS on IE, unlike the Trade Mark (TM) symbol, which announces as "Trade Mark".

Repro:
1. Traverse the page using the virtual cursor until the content with the service mark symbol is encountered.

### Expected result
1. JAWS should announce "My Service Service Mark".

### Actual result
1. JAWS announces "My Service".

### Example

<p>
    My Service <span>&#8480;</span>
</p>

#### JAWS version and build number
JAWS v 2018.1804.26 ILM

 
#### Operating System and version
Win 7 Professional SP1 64-bit

#### Browser and version:
IE 11.0.9600.19002

**Comment by stevefaulkner** [comment]
wontfix
